docs/BUILD.md

```diff
@@ -26,6 +26,39 @@ To build a JAR that can run stand-alone without any additional classes on the cl
 ./gradlew shadowJar
 ```
 
+## Packaging as Installable Application
+
+To build an installable application package appropriate for your operating system
+(e.g. Windows, Mac OS, Linux), first make sure you have a recent version of the
```

Comment:
I think that since the Mac OS application doesn't work at the moment (I assume the same for Linux), we could include a note saying that the installable application can be built on any OS, but only works on Windows at the moment. It could be here or in the `README.md`.

Reply:
I added a note in 5e02c89531fd662b202bdb716f18d4772c533bad